### DISCUSSION TITLE
basic documentation?

### DISCUSSION THREAD

**Discussion by don-ferris** [chipzoller/hugo-clarity#374] [Q&A]
Are there docs somewhere to help noobs figure this out? I got my first post up but I'm struggling to do much else. I can't change the theme colors, can't change (or even see) the nav menu/additional pages. What I want is basically what's shown [here](https://raw.githubusercontent.com/chipzoller/hugo-clarity/master/images/screenshot.png) - the logo, menu bar, and social media links, along with some color and font size changes - but I have no idea how to get there from where I am. I found the docs for Hugo but I assume that it's not going to cover (adequately or at all) the theme changes I want to make in hugo-clarity.

**Comment by chipzoller** [comment]
Color changes we don't provide easily-adjustable knobs from what I recall. Everything else should be fairly straightforward. That's one of the reasons we included that sample site...where you can clone it and drop-in your own replacements. If you have specific areas of improvement we're certainly open (including PRs).

**Reply by don-ferris** [reply]
Brother, it's not straightforward to me. And I've done numerous websites with a simple text editor as my IDE of choice. I'm good with HTML and CSS; I'm not looking for color knobs. I've searched through the documentation I could find, created /assets/sass/_override.sass with 
```
body {
    background: #333333;
}

.nav_header {
    background-color: #222222;
}

h2 {
    font-size: 150%;
}
```
but nothing changed. I created variables.sass in the same directory and copied/pasted the code from your repo but nothing in there made much sense to me so I left it alone. I looked in exampleSite - didn't see a sass folder or file I could emulate anywhere.

Once I get this up and working (and get a couple other small projects done), I'd be happy to create some docs for this. ('Cause I'm having trouble with some other stuff, too, and I can't find anything that tells me what to do. Pretty sure I found a bug, too, but I could just be doing something wrong.)

**Comment by don-ferris** [comment]
Disregard. I'm just going to setup a Wordpress blog.
